server/views/macros/punishmentRadios.njk

```diff
@@ -81,59 +97,71 @@
     }) }}
   {% endset -%}
 
-  {% set radioButtonItems =  [
-    {
-    value: "PRIVILEGE",
-    text: "Loss of privileges",
-    checked: punishmentType == 'PRIVILEGE',
-    conditional: { html: privilegeHtml }      
-    },
-    {
-    value: "EARNINGS",
-    text: "Stoppage of earnings",
-    checked: punishmentType == 'EARNINGS',
-    conditional: { html: stoppageOfEarningsHtml }      
-    },
-    {
-    value: "CONFINEMENT",
-    text: "Cellular confinement",
-    checked: punishmentType == 'CONFINEMENT'
-    },
+  {% set radioButtonItems = [
     {
-    value: "REMOVAL_ACTIVITY",
-    text: "Removal from activity",
-    checked: punishmentType == 'REMOVAL_ACTIVITY'
-    },
-    {
-    value: "EXCLUSION_WORK",
-    text: "Exclusion from associated work",
-    checked: punishmentType == 'EXCLUSION_WORK'
-    },
-    {
-    value: "EXTRA_WORK",
-    text: "Extra work",
-    checked: punishmentType == 'EXTRA_WORK'
-    },
-    {
-    value: "REMOVAL_WING",
-    text: "Removal from wing or unit",
-    checked: punishmentType == 'REMOVAL_WING'
+      value: "PRIVILEGE",
```

Comment:
Did you change the orderings of the existing radios? or is GitHub showing a funny diff?